pyuvdata/uvdata.py

```diff
@@ -3817,6 +3826,27 @@ def read(self, filename, axis=None, file_type=None, allow_rephase=True,
         correct_cable_len : bool
             Flag to apply cable length correction. Only used if file_type is
             'mwa_corr_fits'.
+        flag_init: bool
+            Set to True in order to do routine flagging of coarse channel edges,
+            start or end integrations, or the center fine channel of each coarse
+            channel. See associated keywords. Only used if file_type is
+            'mwa_corr_fits'.
+        edge_width: float
+            Only used if flag_init is True. The width to flag on the edge of
+            each coarse channel, in hz. Errors if not equal to integer multiple
+            of channel_width. Set to 0 for no edge flagging. Only used if
+            file_type is 'mwa_corr_fits'.
+        start_flag: float
+            Only used if flag_init is True. The number of seconds to flag at the
+            beginning of the observation. Set to 0 for no flagging. Only used if
+            file_type is 'mwa_corr_fits'.
+        end_flag: floats
+            Only used if flag_init is True. The number of seconds to flag at the
+            end of the observation. Set to 0 for no flagging. Only used if
+            file_type is 'mwa_corr_fits'.
+        flag_dc_offset: bool
+            Only used if flag_init is True. Set to True to flag the center fine
+            channel of each coarse channel. Only used if file_type is 'mwa_corr_fits'.
```

Comment:
I'm not sure these descriptions are sufficient for non MWA users. Maybe adding something to the first sentence saying they're specific to the MWA? Or maybe changing the parameter name in the `read` method to make it clear they are MWA specific @mkolopanis @plaplant would love your input here.

Reply:
What if I switch the first and last sentences?

Reply:
I was thinking yesterday it would be nice if we could partition the parameters sections by like `generic`, `uvfits`, `miriad`, etc specific parameter sections. Numpy docstyle does not seem to sepcifically allow that, or disallow that though. There is only `parameters` and `other parameters`. The latter reserved for "lesser used parameters". Does not seem to fit the need.

Reply:
I think the order of the parameters can be re-arranged and that might help. I'm ok with adding section headings too, we'd need to do some testing to make sure they render properly with sphinx.

Reply:
I have combined the first and last sentence, putting the last sentence first so that it is clear that the keyword is MWA-specific.